### ISSUE TITLE
Evaluation on custom dataset KITTI-like

### ISSUE THREAD

**Issue by gcaldarulo7** [JonathonLuiten/TrackEval#74]
Hi
I'm trying to evaluate tracking results on a custom dataset which has been converted to the KITTI 2d format.
I downloaded the example data for some clarity on the structure the data should have but I can't figure out what the evaluate_tracking.seqmap.test file is representing and how I should configure it for my custom dataset.
Hope anyone can help on this, 
thanks in advance

**Comment by callmesora** [comment]
Have you figured this out? I'm having the same problem

**Comment by gcaldarulo7** [comment]
Yes, it's pretty striaghtforward actually, all you need to change is putting the exact number of frames for each of the sequences in your dataset. So if you have 3 sequence with x,y,z frames respectively all you need to put in the file is something like
0000 empty 000000 00000x
0001 empty 000000 00000y
0000 empty 000000 00000z

Hope this helps!

**Comment by gcaldarulo7** [comment]
The evaluation algorithm is very easy to use, you can refer to the run_kitti.py script, but honestly all I did after having my tracker output and ground truth data in the kitti format was running said script with the argument --SPLIT_TO_EVAL test as I wanted to evaluate the testing split. Of course you need your ground truth to be in the kitti format aswell, which is the same format used for the outputs of the trackers

**Comment by dodeokwangyooo** [comment]
> Yes, it's pretty striaghtforward actually, all you need to change is putting the exact number of frames for each of the sequences in your dataset. So if you have 3 sequence with x,y,z frames respectively all you need to put in the file is something like 0000 empty 000000 00000x 0001 empty 000000 00000y 0000 empty 000000 00000z
> 
> Hope this helps!

wow i have the same problem ,  i still can't understand the file like 0000 empty 000000 000000x . can u give me a zip?

**Comment by dodeokwangyooo** [comment]
hey i have some problems，can you teach me how to submit my results to the kitti ? I want to get the test results . you know the test sequence of 1 to 28.
